tests/plugins/__init__.py

```diff
@@ -17,15 +17,6 @@
 
 import unittest
 
-from troubadix.helper.patterns import (
-    ScriptTagPatterns,
-    SpecialScriptTagPatterns,
-)
-
 
 class PluginTestCase(unittest.TestCase):
-    def setUp(self) -> None:
-        self.tag_pattern = ScriptTagPatterns().pattern
-        self.special_tag_pattern = SpecialScriptTagPatterns().pattern
-
-        return super().setUp()
+    pass
```

Comment:
Pretty obsolete Class now ... Remove it again?

Reply:
Yes at the moment. But I have some things in mind for making the plugin tests easier to write and therefore I kept the class.

Reply:
Ok cool!